web/src/features/menu/context/Item.tsx

```diff
@@ -66,13 +67,15 @@ const Item: React.FC<{
                   }}
                 />
               )}
-              <Box>
+              <Box w="100%">
                 <Box paddingBottom={button.description ? 1 : 0}>
                   <Text w="100%" fontWeight="medium" color={button.disabled ? '#718096' : undefined}>
                     {button.title ? button.title : buttonKey}
                   </Text>
                 </Box>
-                {button.description && (
```

Comment:
Why not make it so you can have a progress bar *and* a description as well? 🤷

Reply:
I thought it would bloat too much, but adjusted it to support if people want it!